example/src/blocks/multiple-image-example/edit.js

```diff
@@ -0,0 +1,25 @@
+import { __ } from '@wordpress/i18n';
+import { useBlockProps } from '@wordpress/block-editor';
+
+import { Image } from '@10up/block-components';
+
+export function BlockEdit(props) {
+    const {
+        attributes,
+        setAttributes
+    } = props;
+
+    const { image1, image2, image3, focalPoint1, focalPoint2, focalPoint3 } = attributes;
+    const blockProps = useBlockProps();
+
+    return (
+        <div {...blockProps}>
+
+            <Image id={image1} size="large" onSelect={(image) => setAttributes({image1: image.id })} className="example-image" focalPoint={focalPoint1} onChangeFocalPoint={(value) => setAttributes({focalPoint1: value})} hasInlineControls={true} onRemove={() => setAttributes({image1: null})} isOptional={false} />
```

Comment:
Could you refactor this to have linebreaks between the individual attributes? That would make it much more readable :)